CMakeLists.txt

```diff
@@ -41,14 +50,18 @@
 # file has changed.
 
 
-cmake_minimum_required (VERSION 2.6)
+cmake_minimum_required (VERSION 3.1.0 FATAL_ERROR)
```

Comment:
Can this be made conditional on building QtSLiM?  The BSCB cluster that I use at Cornell, for example, is still on cmake 2.8.12.2, and I'm sure there are lots of other such cases; institutions are often very slow to upgrade.

Reply:
Yes, that can be an issue, I will try moving the higher version in the IF.

Reply:
Moved to the IF.